packages/helm-charts/odis/templates/signer-ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: odis-signer-ingress
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: nginx
+spec:
+  tls:
+  - secretName: {{ .Release.Namespace }}-web-tls
```

Comment:
Just curious, what is this secret for?

Reply:
Nice catch.  That's not needed, and I'll remove it.  It's just a remnant of my copying of the komenci helm chat (which it also doesn't appear to be needed for that helm chart).

Having said that, this does bring up a potential issue.  Since the signer's are exposed via an IP address, their respective frontdoors will need to connect to the k8s clusters via http, instead of https.  When it connects to https, it wouldn't have a valid TLS/SSL cert.

To support https, I'll need to create an entry in the clabs DNS table (within GCP) and setup TLS/SSL certificate handling.

Let me know if this is needed, and I can go ahead and do that.